### PR TITLE
Array-Game-Winner-Rounds

### PR DESCRIPTION
**Pull request by rimesta21**
Adding a solution to LeetCode challenge 1535. Find the Winner of an Array Game solved 10/16/20. This problem was introduced by Daniel G. Shanker.

### CODE REVIEW

**Comment by xm94** [comment]
Hey @rimesta21 ! Sorry for just now noticing this, it looks like you've accidentally pushed some IDE files, I can merge this in once you clear those up!

File: Array-Game-Rounds-Winner/src/main.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        return winner;
+    }
+
+    public static void main(String[] s) {
+        int[] arr = {2,1,3,5,4,6,7};
+        System.out.println(arrayGame(arr, 2));
+        //Expected output: 5
+
+    }
+}
```
Comment (xm94) [inline Array-Game-Rounds-Winner/src/main.java:26]: So pretty much make the PR just contain this file, make sure to be careful with what you add when commiting

**Review by xm94 — CHANGES_REQUESTED** [review]

**Comment by rimesta21** [comment]
Hey, @xm94 so sorry about that! I think I have deleted all the files besides main. I will be more careful next time.

**Review by xm94 — APPROVED** [review]
Awesome, sounds good

**Pull request merged**